L_ZWay2.lua

```diff
@@ -1280,11 +1295,32 @@ local function configureDevice (id, name, ldv, child)
 --    name = "rgb #" .. id
 
   elseif ((classes["37"] and #classes["37"] == 1)             -- ... just one switch
-  or      (classes["38"] and #classes["38"] == 1) )           -- ... OR just one dimmer
-  and not classes["49"] then                                  -- ...but NOT any sensors
-    local v = (classes["38"] or empty)[1] or classes["37"][1]  -- then go for the dimmer
-    upnp_file, json_file, name = add_updater(v)
-
+  or      (classes["38"] and #classes["38"] == 1) ) then         -- ... OR just one dimmer
+    if not classes["49"] and not classes ["113"] then             -- ...but NOT any sensors
+      local v = (classes["38"] or empty)[1] or classes["37"][1]
+      upnp_file, json_file, name = add_updater(v)
+    else
+      local cl = (classes["38"] or empty)[1] or classes["37"][1]
+      local meta = cl.meta
+      name = table.concat {"multi #", meta.node, '-', meta.instance}
+      local types = {}
+      for _, v in ipairs (cl) do
+          types["Switch"] = (types["Switch"] or 0) + 1
+        child[v.meta.altid] = true                            -- force child creation
+      end
```

Comment:
This looks suspect to me, since it seems that `cl` is a single `vDev` and not a list of `vDev`s?

Maybe you mean:
```lua
local cl = classes["38"] or classes["37"]
```